frontend/src/components/pages/CreateReview/BookModal.tsx

```diff
@@ -41,7 +43,14 @@ const kMaxPrice = 1000;
  * Modal for user to input new book for review
  * */
 const BookModal = (props: BookModalProps): React.ReactElement => {
-  const { isOpen, onClose, booksAdded, handleBooksAdded } = props;
+  const {
```

Comment:
Might be a good idea to create an interface for the props (with documentation), like so: https://github.com/uwblueprint/ccbc/blob/development/frontend/src/components/PreviewReviewModal.tsx#L18